winternlc/create/make_linearpoly_corrections.py

```diff
@@ -0,0 +1,293 @@
+import os
+import sys
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+import numpy as np
+from utils import extract_pixel_values, find_median_files, get_exposure_time
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../")))
+from config import DEFAULT_CUTOFF, output_directory, test_directory
```

Comment:
Can you just use `from winternlc.config import ....`? The sys.path stuff is not considered good practice for python.